documentation/src/main/resources/pages/ditto/basic-conditional-requests.md

```diff
@@ -160,6 +160,154 @@ was introduced.
 Upon activation, the digital twin stored in Eclipse Ditto will implicitly be updated with the latest data from the 
 _live response_ sent by the device.
 
+## Path-specific conditions (Since 3.8.0)
+
+In addition to global conditions, Ditto supports path-specific conditions for merge operations using the `patch-conditions` header. This feature allows you to apply different RQL conditions to different parts of a merge patch, enabling fine-grained control over which parts of the patch are applied based on the current state of the Thing.
```

Comment:
Hm, maybe we should name the header `merge-thing-patch-conditions` to already make clear in the header that only the `MergeThing` command supports that ..

What do you think @hu-ahmed ?